src/main/java/mate/jdbc/service/ManufacturerServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Manufacturer;
+
+@Service
+public class ManufacturerServiceImpl implements ManufacturerService {
+    @Inject
+    private ManufacturerDao manufacturerDao;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        return manufacturerDao.create(manufacturer);
+    }
+
+    @Override
+    public Manufacturer get(Long id) {
+        return manufacturerDao.get(id).get();
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        return manufacturerDao.getAll();
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        return manufacturerDao.update(manufacturer);
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        return manufacturerDao.delete(id);
+    }
+}
```

Comment:
Need to create init_db.sql file with statements of tables

Reply:
It is not necessary, but I'm greatful for your suggestion